src/state/persisted/legacy.ts

```diff
@@ -7,15 +7,15 @@ import {write, read} from '#/state/persisted/store'
 /**
  * The shape of the serialized data from our legacy Mobx store.
  */
-type LegacySchema = {
+export type LegacySchema = {
   shell: {
     colorMode: 'system' | 'light' | 'dark'
   }
   session: {
     data: {
       service: string
       did: `did:plc:${string}`
-    }
+    } | null
```

Comment:
Small type update. Access below was using optional chaining, but this is more correct.